backend/src/v5/schemas/tickets/index.js

```diff
@@ -272,7 +272,7 @@ Tickets.processReadOnlyValues = (oldTicket, newTicket, user) => {
 	}
 };
 
-const uuidString = Yup.string().transform((val, orgVal) => UUIDToString(orgVal));
+const uuidString = Yup.string().transform((val, orgVal) => UUIDToString(orgVal)).nullable();
```

Comment:
is this really correct? You're imposing nullable on everything else that was using this to begin with.

And why would we antipate this to be nullable? surely we don't want to return `[null]` as the  image list?

Reply:
the reason is that when a ticket is updated and an image prop is set to null there is an error thrown here

![image](https://github.com/user-attachments/assets/cd49bbde-895d-4e40-aeee-837ee4007cad)

Reply:
I changed it now and added nullable to the specific props